circle.yml

```diff
@@ -0,0 +1,36 @@
+machine:
```

Comment:
it looks like the tests didn't run on this PR - do you need help setting them up?